### DISCUSSION TITLE
Vaultwarden update

### DISCUSSION THREAD

**Discussion by tomdh76** [tteck/Proxmox#88] [Q&A] [answered]
I have a working LXC container with vaultwarden. I only wonder if vaultwarden will be updated if this is easy or do I need to build the whole container again? And what do you use as a backup tool for backuping up the dbsqlite?

**Comment by tteck** [comment]
Why not use the update script?

**Reply by tomdh76** [reply]
> Why not use the update script?

I missed that script, nice !

**Comment by tteck** [comment, marked as answer]
If you want to backup data
```yaml
cp -R /var/lib/vaultwarden/data data-backup
```
If you want to restore data
```yaml
cp -R data-backup/* /var/lib/vaultwarden/data
```
